:sparkles: Add points (#7)
zenika-open-source/how-to-promote-my-open-source: README.md

- [English](./README.md)
- [Français](./README-fr.md)

<br />

---

<br />

## 1. 🎢 Preparation step

<br />

<details>
<summary>😎 Choose a cool name for your project</summary>
<p>

> Choose a name users can easily remember. It must not necessarly includes keywords related to the technology you're using within your project (ex: [bento-starter](https://github.com/kefranabg/bento-starter)). 

</p>
</details>

<details>
<summary>💅 Make your README pretty</summary>
<p>

> README is the first your user will see. Make it simple, pretty and easy to read. [Here is a list of beautiful READMEs](https://github.com/matiassingers/awesome-readme).

</p>
</details>

<details>
<summary>💪 Highlight strong points of your project </summary>
<p>

> Identify your project strengths and make sure that visitors see them at first.

</p>
</details>

<details>
<summary>✨ Provide a demo of your project</summary>
<p>

> README is the first your user will see. Make it simple, pretty and easy to read. [Here is a list of beautiful READMEs](https://github.com/matiassingers/awesome-readme).

 </p>
</details>

<details>
<summary>👌 Insall/Usage must be super easy</summary>
<p>

> > Install/Usage must be as simple as possible. You'll probably lost users if your project is not user friendly.

Reviewer: ```suggestion
> You'll probably lost users if your project is not user friendly.
```

 </p>
</details>

<details>
<summary>📘 Create a clean and well structured documentation</summary>
<p>

> > Create a good documentation is probably the most important step. If your have a small documentation, you can include it in your README. Otherwise, you should probably host it in a diferrent website. Projects like [vuepress](https://v1.vuepress.vuejs.org) helps you creating clean documentations in a simple way.

Reviewer: ```suggestion
> It is probably the most important step. If your have a small documentation, you can include it in your README. Otherwise, you should probably host it in a different website. Projects like [vuepress](https://v1.vuepress.vuejs.org) helps you creating clean documentations in a simple way.
```

 </p>
</details>

<br />

## 2. 📢 Spread the word about your hard works

<br />

<details>
<summary>⭐ Talk about your project to your friends and colleagues to get your first github stars before publishing to social networks</summary>
<p>

> A minimum stars amount make your project more trustable than a zero star project. Ask people you know to support your project before doing a public annouccement on social medias.

</p>
</details>

<details>

> <summary>↗️ Share the project on social medias an specialized platforms</summary>

Reviewer 2: Don't know why, but the emoji doesn't display well here on README.md

![type](https://user-images.githubusercontent.com/53863748/63269952-7167f780-c297-11e9-82c2-867e53694980.PNG)

<p>

> Tell the world about your awesome project! Publish on social medias and specialized platforms:

- [Twitter](https://twitter.com)
- [Linkedin](https://www.linkedin.com/)
- [Facebook](https://www.facebook.com/)
- [Reddit](https://www.reddit.com/)
- [Dev.to](https://dev.to/)
- [Lobsters](https://lobste.rs/)
- [Hacker News](https://news.ycombinator.com/)
- [Product Hunt](https://www.producthunt.com/)
- [Beta page](https://betapage.co/)
- [Human Coders](https://news.humancoders.com/)
</p>
</details>

<details>
<summary>📃 Write articles that mention your project</summary>
<p>

> > Write articles to talk about your project (purpose can be the technical stack you used, how your project works, problems you encountered, etc). Publish your articles to [medium](https://medium.com/) or [dev.to](https://dev.to/).

Reviewer: ```suggestion
> Write articles about your project (purpose can be the technical stack you used, how your project works, problems you encountered, etc). Publish your articles to [medium](https://medium.com/) or [dev.to](https://dev.to/).
```

</p>
</details>

<details>

> <summary>🎤 Present your project at conferences/meetups</summary>

Reviewer: ```suggestion
<summary>🎤 Present your project at conferences/meetups/podcast</summary>
```
